src/tokenized-line.coffee

```diff
@@ -111,27 +111,32 @@ class TokenizedLine
 
       return maxColumn
 
-  # Calculates how many trailing spaces in this line's indentation cannot fit in a single tab.
+  # For a given `indentLevel`, calculates how many trailing spaces cannot fit in a single tab.
   #
-  # Returns a {Number} representing the odd indentation spaces in this line.
-  getOddIndentationSpaces: ->
-    oddIndentLevel = @indentLevel - Math.floor(@indentLevel)
+  # indentLevel - {Number}
+  # Returns a {Number} representing the odd indentation spaces in `indentLevel`.
+  oddIndentationSpacesForIndentLevel: (indentLevel) ->
+    oddIndentLevel = indentLevel - Math.floor(indentLevel)
     Math.round(@tabLength * oddIndentLevel)
 
-  buildSoftWrapIndentationTokens: (token) ->
-    indentTokens = [0...Math.floor(@indentLevel)].map =>
+  buildSoftWrapIndentationTokens: (token, hangingIndent) ->
+    hangingIndentLevel = hangingIndent / @tabLength
+    indentLevel = @indentLevel + hangingIndentLevel
+    indentTokens = [0...Math.floor(indentLevel)].map =>
       token.buildSoftWrapIndentationToken(@tabLength)
 
-    if @getOddIndentationSpaces()
+    if oddIndentationSpaces = @oddIndentationSpacesForIndentLevel(indentLevel)
       indentTokens.concat(
-        token.buildSoftWrapIndentationToken @getOddIndentationSpaces()
+        token.buildSoftWrapIndentationToken(oddIndentationSpaces)
       )
     else
       indentTokens
 
-  softWrapAt: (column) ->
+  softWrapAt: (column, options = {}) ->
```

Comment:
What do you think about just passing the overhang in as a direct parameter rather than using an options object. It's not as pretty but it saves an object allocation and since this code is internal, seems like a reasonable trade-off.